src/app-interoperability/application-package-definition.linkml.yaml

```
@@ -0,0 +1,497 @@
+        rank: 60
+        range: string
+
+  Parameter:
+    description: Defines a configurable parameter for the application.
+    rank: 90
+    attributes:
+      name: #Arne: error in spec. This attribute is missing.
+        description: Name of the parameter.
+        rank: 10
+        identifier: true
+        required: true
+        range: string
+      value:
+        description: >-
+          The parameter's default value. 
+          Accepted data types are string, integer, double, boolean, array[string], array[integer], array[double], array[boolean].
+        rank: 20
+        range: string #Arne: error in spec. type not properly given.
+      targets:
+        description: >-
+          Used to indicate which component the value should be applied to when installing, or updating, the application. 
+          See the [Target](#target-attributes) section below.
+        rank: 30
+        range: Target
+        required: true
+        multivalued: true
+
+  Target:
+    description: Specifies where the parameter applies in the deployment.
+    rank: 100
+    attributes:
+      pointer:
+        description: >-
+          The name of the parameter in the deployment configuration. 
+          For Helm deployments, this is the dot notation for the matching element in the `values.yaml` file. This follows the same naming convention you would use with the `--set` command line argument with the `helm install` command. 
+          For docker-compose deployments, this is the name of the environment variable to set.
+        rank: 10
+        range: string
+        required: true
+      components:
+        description: >-
+          Indicates which deployment profile [component](#component-attribute) the parameter target applies to. 
+          The component name specified here MUST match a component name in the [deployment profiles](#deployment-profiles-attributes) section.
+        rank: 20
+        range: string
+        multivalued: true
+        required: true
+
+  Configuration:
+    description: Configuration layout and validation rules.
+    rank: 110
+    attributes:
+      sections:
+        description: >-
+          Sections are used to group related parameters together, 
+          so it is possible to present a user interface with a logical grouping of the parameters in each section. 
+          See the [Section](#section-attributes) section below.
+        rank: 10
+        range: Section
+        multivalued: true
+        required: true
+      schema:
+        description: >-
+          Schema is used to provide details about how to validate each parameter value. 
+          At a minimum, the parameter value must be validated to match the schema's data type. 
+          The schema indicates additional rules the provided value must satisfy to be considered valid input. 
+          See the [Schema](#schema-attributes) section below.
+        rank: 20
+        range: Schema
+        multivalued: true
```

@arne-broering schema is an array/list

```suggestion
        multivalued: true
        inlined: true
        inlined_as_list: true
```